scripts/deploy.sh

```diff
@@ -30,6 +30,11 @@ function run_ansible {
 # begin the bootstrap process
 cd ${OA_DIR}
 
+# bootstrap ansible and install galaxy roles (needed whether AIO or multinode)
+which openstack-ansible || ./scripts/bootstrap-ansible.sh
+ansible-galaxy install --role-file=/opt/rpc-openstack/ansible-role-requirements.yml --force \
+                           --roles-path=/opt/rpc-openstack/rpcd/playbooks/roles
+
```

Comment:
osa itself installs some roles via galaxy, but it allows them to be installed into the default location of ```/etc/ansible/roles```. Is there any reason why we couldn't just do the same so all 'galaxy installed' roles are in the same location.